packages/@tinacms/cli/src/cmds/init/configure.ts

```diff
@@ -0,0 +1,446 @@
+import { Framework, GeneratedFile, InitEnvironment } from '.'
+import prompts, { PromptType } from 'prompts'
+import { linkText, logText } from '../../utils/theme'
+import crypto from 'crypto-js'
+import fs from 'fs-extra'
+import { logger } from '../../logger'
+
+async function writeTinaEnv(
+  config: Record<any, any>,
+  opts: { isSelfHosted: boolean }
+) {
+  let envFile = `// Generated by Tina CLI\n\n`
+
+  if (opts.isSelfHosted) {
+    if (config.kvRestApiUrl !== undefined) {
+      envFile += `KV_REST_API_URL=${config.kvRestApiUrl}\n`
+    }
+    if (config.kvRestApiToken !== undefined) {
+      envFile += `KV_REST_API_TOKEN=${config.kvRestApiToken}\n`
+    }
+    if (config.nextAuthSecret !== undefined) {
+      envFile += `NEXTAUTH_SECRET=${config.nextAuthSecret}\n`
+    }
+    if (config.vercelKVNextAuthCredentialsKey !== undefined) {
+      envFile += `NEXTAUTH_CREDENTIALS_KEY=${config.vercelKVNextAuthCredentialsKey}\n`
+    }
+    if (config.githubToken !== undefined) {
+      envFile += `GITHUB_PERSONAL_ACCESS_TOKEN=${config.githubToken}\n`
+    }
+  }
+
+  if (config.clientId !== undefined) {
+    envFile += `NEXT_PUBLIC_TINA_CLIENT_ID=${config.clientId}\n`
+  }
+
+  if (config.token !== undefined) {
+    envFile += `TINA_TOKEN=${config.token}\n`
+  }
+  await fs.writeFile('.env.tina', envFile)
+}
+
+async function configure(
+  env: InitEnvironment,
+  opts: { debug?: boolean; isBacked?: boolean }
```

Comment:
@logan-anderson is this supposed to be `isBackend`?

Reply:
😅 Yes. Good catch

Reply:
Fixed!